tests/ioc/test_file_server.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+import json
+import os
+from typing import Dict
+
+import aiohttp
+import pytest
+from aiohttp.test_utils import TestClient, TestServer
+
+from pandablocks.ioc.ioc import initialise_server
+from tests.conftest import DummyServer
+
+# Tests for the bobfile server
+
+test_bob_file_dict: Dict[str, str] = {"TEST.bob": "<Bobfile>Contents</Bobfile>"}
```

Comment:
It's worth turning all instances of `"TEST.bob"` into a constant. Same applies to the content of the file. This makes modifying/extending tests in the future a bit nicer.